source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import GroupedList from '../../molecules/GroupedList/GroupedList';
+import SummaryListItem from './SummaryListItem';
+
+export interface Item {
```

Comment:
Would like to se a prefix on this ie ISummerListItem, setting I at the begining of an Interface seems to be a common pattern in the typescript community (y)

Reply:
Renamed to SummaryListItem